src/OrbitGl/TimerTrack.h

```diff
@@ -95,9 +95,11 @@ class TimerTrack : public Track {
 
   [[nodiscard]] bool IsCollapsible() const override { return depth_ > 1; }
 
-  virtual void UpdateBoxHeight();
+  [[nodiscard]] virtual float GetBoxHeight() const { return layout_->GetTextBoxHeight(); }
   [[nodiscard]] virtual float GetBoxHeight(
-      const orbit_client_protos::TimerInfo& /*timer_info*/) const;
+      const orbit_client_protos::TimerInfo& /*timer_info*/) const {
+    return GetBoxHeight();
+  }
```

Comment:
Yes, I also had a strange feeling here. I will try to rename GetBoxHeight() to another name because both of them are used in FrameTrack for example and it's error prone as you said, thanks!

Reply:
Agree that we should rename the new `GetBoxHeight()` method. 

The `TimerTrack::GetBoxHeight(const TimerInfo&)` and `TimerTrack:GetBoxHeight()` return values for different purpose. Hence using the same name seems to be confusing.

Reply:
After discussed offline with Henning, we agree that we need both function anyway (unless we modified FrameTrack to have its own UpdatePrimitives) and will be renamed to GetDynamicBoxHeight and GetDefaultBoxHeight